### PR TITLE
Update release-announcement.md

### PR DESCRIPTION
**Pull request by amyblais**
Edits based on https://docs.google.com/document/d/1JRSu8QZIMIQZRUaocvbNT_NDquziH2P8cTBsBnTf2JE/edit?ts=604637fd

### CODE REVIEW

File: operations/research-and-development/product/how-to-guides-for-product/release-announcement.md [hunk 1]
```diff
@@ -14,9 +14,35 @@ It's important to think of the release announcement as a press release. We're co
...
+
+* A draft of the release announcement and tweets are completed by Product Managers. Product Managers prepare the release announcement from their 2-pager documents.
+
+#### C. \(T-minus 8 working days\) Lead Product Managers' Review
+
+* Jason and Katie do a final review on the release announcement before handing it off to Jack.
+
+#### D. \(T-minus 7 working days\) Product Marketing Director Review
+
+* The PM-approved draft will be provided for review to the Product Marketing Director.
```
Comment (jasonblais) [inline operations/research-and-development/product/how-to-guides-for-product/release-announcement.md]:
```suggestion
* The PM-approved draft will be provided for review to the Director of Product Marketing.
```

File: operations/research-and-development/product/how-to-guides-for-product/release-announcement.md [hunk 2]
```diff
@@ -14,9 +14,35 @@ It's important to think of the release announcement as a press release. We're co
...
 
-Release announcements and tweets are due from Product Managers at T-14, so a first draft can be completed by T-11, per our [release announcement process](operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements). They should be provided to your Lead (Katie), who will coordinate all into one single document for the release. A final draft of the combined release announcement for the newsletter will be provided for review to CPO and MLT team by T-10. Finally, the approved release announcement will be provided to marketing by T-2 and published on release day.
+#### A. \(T-minus 14 working days\) Outline
+
+* Release Manager provides an outline of the release announcement and deadlines for Product Managers, along with a heads-up for Jack that the blog post is coming.
```
Comment (jasonblais) [inline operations/research-and-development/product/how-to-guides-for-product/release-announcement.md]:
```suggestion
* Release Manager provides an outline of the release announcement and deadlines for Product Managers, along with a heads-up for Director of Product Marketing that the blog post is coming.
```

File: operations/research-and-development/product/how-to-guides-for-product/release-announcement.md [hunk 3]
```diff
@@ -14,9 +14,35 @@ It's important to think of the release announcement as a press release. We're co
...
+
+#### B. \(T-minus 9 working days\) First draft
+
+* A draft of the release announcement and tweets are completed by Product Managers. Product Managers prepare the release announcement from their 2-pager documents.
+
+#### C. \(T-minus 8 working days\) Lead Product Managers' Review
+
+* Jason and Katie do a final review on the release announcement before handing it off to Jack.
```
Comment (jasonblais) [inline operations/research-and-development/product/how-to-guides-for-product/release-announcement.md]:
```suggestion
* Product Management leads do a final review on the release announcement before handing it off to Director of Product Marketing.
```

**Review by jasonblais — CHANGES_REQUESTED** [review]
Just minor proposals, else looks good to me!

**Review by wiersgallak — DISMISSED** [review]
Thanks, Amy!

**Review by jasonblais — APPROVED** [review]

**Pull request merged**